.github/workflows/s3-tests.yml

```diff
@@ -0,0 +1,318 @@
+name:  S3 tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+      - support/**
+    types: [opened, synchronize]
+    paths-ignore:
+      - '**/*.md'
+  release:
+    types:
+      - published
+  workflow_dispatch:
+    inputs:
+      neofs-s3-gw_ref:
+        description: 'neofs-s3-gw ref. Default ref - latest master. Examples: v0.27.0, 8fdcc6d7e798e6511be8806b81894622e72d7fdc, branch_name'
```

Comment:
This will not work, only the master will always run.
Yes, the neofs-node project has this bug too.

Reply:
I created a bug:
https://github.com/nspcc-dev/neofs-node/issues/2509